terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1799,7 +1799,9 @@ def qq(*words):
                             '-a', 'fetch-config',  # action (fetch file from location specified in -c)
                             '-c', 'file:/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json',
                             '-s'  # restart agent afterwards
-                        ]
+                        ],
+                        ['yum', '-y', 'update'],
+                        ['reboot']
```

Comment:
This is already done in line 1771. Was this also tested?

Reply:
> Was this also tested?

Confirmed `yum -y update` was run automatically as part of the deploy by deploying `dev.gitlab`, then connecting to the instance and manually running `yum update`. No updates were needed.